### ISSUE TITLE
Initial example not working

### ISSUE THREAD

**Issue by lukemovement** [run-llama/LlamaIndexTS#1114]
Version: 0.5.15
OS: Ubuntu
Node: 0.5.15

```ts
import { VectorStoreIndex } from "llamaindex";

// Load essay from abramov.txt in Node
const essay =
  'The Normans (Norman: Nourmands; French: Normands; Latin: Normanni) were the people who in the 10th and 11th centuries gave their name to Normandy, a region in France. They were descended from Norse ("Norman" comes from "Norseman") raiders and pirates from Denmark, Iceland and Norway who, under their leader Rollo, agreed to swear fealty to King Charles III of West Francia. Through generations of assimilation and mixing with the native Frankish and Roman-Gaulish populations, their descendants would gradually merge with the Carolingian-based cultures of West Francia. The distinct cultural and ethnic identity of the Normans emerged initially in the first half of the 10th century, and it continued to evolve over the succeeding centuries.';

// Create Document object with essay
const document = { text: essay };

// Split text and create embeddings. Store them in a VectorStoreIndex
const index = await VectorStoreIndex.fromDocuments([document]);

// Query the index
const queryEngine = index.asQueryEngine();
const response = await queryEngine.query({
  query: "In what country is Normandy located?",
});

// Output response
console.log(response.toString());
```

```sh
file:///home/luke/Personal/ts-ai/node_modules/llamaindex/dist/readers/CSVReader.js:1
import { Document, FileReader } from "@llamaindex/core/schema";
                   ^^^^^^^^^^
SyntaxError: The requested module '@llamaindex/core/schema' does not provide an export named 'FileReader'
    at ModuleJob._instantiate (node:internal/modules/esm/module_job:134:21)
    at async ModuleJob.run (node:internal/modules/esm/module_job:217:5)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async asyncRunEntryPointWithESMLoader (node:internal/modules/run_main:123:5)

Node.js v20.14.0
```

**Comment by himself65** [comment]
oh, I think there is issue in release CI

**Comment by himself65** [comment]
should fixed in 0.5.16
